### PR TITLE
covid-19-services: Add docs about the new find-support form

### PR DESCRIPTION
**Pull request by issyl0**
- Where relevant I changed "both" to "all" or got rid of the word
  entirely as there are now >2 forms.
- Fix some casing of products, like "PaaS" and fix some accidental,
  unnecessary double spacing between sections.
- Move the blurb about the IVR to the vulnerable people section, as
  that's the only thing it's for.

Trello card: https://trello.com/c/3yABBo1i

### CODE REVIEW

File: source/manual/covid-19-services.html.md [hunk 1]
```diff
@@ -38,17 +32,32 @@ message from the NHS, or been advised by their GP to fill in the form.
...
+- [Service](https://find-coronavirus-support.service.gov.uk/urgent-medical-help)
+- [GitHub](https://github.com/alphagov/govuk-coronavirus-find-support)
 
 ## Architecture
 
 Both applications have a similar architecture of:
 
 - **Framework:** Ruby on Rails
-- **Database:** AWS RDS (business volunteering) and DynamoDB (vulnerable people)
+- **Database:** AWS RDS (business volunteering, find support) and DynamoDB (vulnerable people)
```
Comment (JonathanHallam) [inline source/manual/covid-19-services.html.md:60]: Isn't the find support page using postgres?

File: source/manual/covid-19-services.html.md [hunk 2]
```diff
@@ -91,8 +99,9 @@ This is also stored for 2 hours.
 
 ### Data storage
 
-For both applications, when the user submits the form on the "Check
-your answers" page, the application writes user data to the database.
+For all but the find support form, when the user submits the form on
+the "Check your answers" page, the application writes user data to the
+database.
```
Comment (JonathanHallam) [inline source/manual/covid-19-services.html.md:104]: Might be worth adding when the find support page does this, ie. on submission of the final question

File: source/manual/covid-19-services.html.md (hunk 1 above)
Comment (brucebolt) [reply source/manual/covid-19-services.html.md:60]: RDS is AWS's Postgres/MySQL/etc solution

Comment (JonathanHallam) [reply source/manual/covid-19-services.html.md:60]: 👍

**Pull request merged**